src/Command.ts

```diff
@@ -27,6 +27,10 @@ const executors = {
     },
     clear: (invocation: Invocation, args: Array<string>): void => {
         setTimeout(() => invocation.terminal.clearInvocations(), 0);
+    },
+    exit: (invocation: Invocation, args: Array<string>): void => {
+        var app = require('remote').require('app');
+        app.quit();
```

Comment:
And what about making an ``application delegate`` class with the common methods.

Something like this:
```ts
class ApplicationDelegate {
  get activeTerminal();
  removeTerminal(index: int);
}
```
Instead of requiring the main application class every time, we have common methods that will send messages through ``ipc`` to an event handler in the ``Application class``.